docs/configuration.md

```diff
@@ -0,0 +1,114 @@
+# Configuration
+
+[More configuration information is available in the Neon documentation.](https://neongeckocom.github.io/neon-docs/quick_reference/configuration/)
+
+All configuration is done in the `neon.yaml` file. This file is located in the `/home/neon/xdg/neon/neon.yaml` directory by default. After making changes to the file, you must restart the Neon core services in Yacht for the changes to take effect.
+
+Neon core services:
+
+- [neon-gui](https://github.com/NeonGeckoCom/neon_gui)
+- [neon-enclosure](https://github.com/NeonGeckoCom/neon_enclosure)
+- [neon-audio](https://github.com/NeonGeckoCom/neon_audio)
+- [neon-skills](https://github.com/NeonGeckoCom/neoncore)
+- [neon-speech](https://github.com/NeonGeckoCom/neon_speech)
+- [neon-messagebus](https://github.com/NeonGeckoCom/neon_messagebus)
+
+## Common configuration options
+
+Common configurations are found in the `neon.yaml` file.
+
+!!! warning
+    YAML is a whitespace-sensitive language. Be sure to maintain proper indentation when editing the `neon.yaml` file.
+
+Specific configuration changes are also detailed in blog posts by community dev Mike Gray. [Read the blog posts](https://blog.graywind.org/tags/neon/).
```

Comment:
Consider also a link to [Neon docs](https://neongeckocom.github.io/neon-docs/quick_reference/configuration/)